README.md

```diff
@@ -66,7 +66,7 @@ use postal::{Context, InitOptions, ParseAddressOptions};
 let mut ctx = Context::new();
 
 // enable address parsing for this context
-ctx.init(InitOptions{parse_address: true}).unwrap();
+ctx.init(InitOptions{expand_address: true, parse_address: true}).unwrap();
```

Comment:
I think that you want expand_address to be false here, so you don't load those models when just doing a parse.

Reply:
Ah, good catch, thanks!